pyproject.toml

```
@@ -50,10 +50,20 @@ namespaces = false
 basic-pitch = "basic_pitch.predict:main"
 
 [project.optional-dependencies]
+train = [
+	"basic_pitch[tf,test]",
+    "apache_beam",
+    "mirdata",
+    "smart_open",
+    "sox"
+]
 test = [
+	"apache_beam",
 	"coverage>=5.0.2",
+	"mirdata",
 	"pytest>=6.1.1",
 	"pytest-mock",
+	"sox"
```

```suggestion
test = [
	"coverage>=5.0.2",
	"pytest>=6.1.1",
	"pytest-mock",
```

these should only be needed by train.

---

but for test we're using TestPipeline, so apache_beam at least right?

---

Sure, we can keep that, but it should be transitive from the data extra anyway.